libs/model/src/models/sso_token.ts

```diff
@@ -23,19 +12,18 @@ export default (
     'SsoToken',
     {
       id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
-      issued_at: { type: Sequelize.INTEGER, allowNull: true },
-      issuer: { type: Sequelize.STRING, allowNull: true },
-      address_id: { type: Sequelize.INTEGER, allowNull: true },
-      profile_id: { type: Sequelize.INTEGER, allowNull: true },
-      state_id: { type: Sequelize.STRING, allowNull: true },
+      issued_at: { type: Sequelize.INTEGER, allowNull: false },
+      issuer: { type: Sequelize.STRING, allowNull: false },
+      address_id: { type: Sequelize.INTEGER },
```

Comment:
@jnaviask are SSO tokens only applicable to magic addresses? ...seems like it

Should we make address_id a required field? (allowNulls = false) ... looks like we shouldn't have any records where address_id, issued_at, issuer are null ... can't find any in the dev db anyway

Reply:
@Rotorsoft That is correct -- SSO tokens are only used by magic addresses and should always have an address_id.

Reply:
@Rotorsoft actually they're used to link addresses on Discord, but TBD the status of that feature.